### ISSUE TITLE
Has stopped searching

### ISSUE THREAD

**Issue by xrats** [griff/metaz#42]
I haven't been able to get it to search for titles for the last week or so.  Please fix!  Thank you.

**Comment by griff** [comment]
Has all searching stopped working or is it only tagChimp results that are missing?

And have you made sure that the video type is correctly set to either Movie or TV Show?

**Comment by xrats** [comment]
Well I guess it's searching, just the TagChimp results aren't showing up.  I didn't change any settings, it was working one day, not the next.  Thanks.

**Comment by xrats** [comment]
Sorry, you're right.  I never touched the settings, but somehow it got switched to Home Movies, I switched it back to Movies and it seems to be working again.  Thanks for your help and for the awesome program!

**Comment by griff** [comment]
tagChimp is down because of too much traffic: http://www.facebook.com/tagChimp/posts/119809048184558

Of course it would be nice if MetaZ could show errors like this to the users instead of just not returning anything. So I have created a separate issue for that: #43 

In iTunes 11 the video type value that was used for Movie became Home Movie and so in the latest version of MetaZ this was changed there as well. This had the unfortunate side effect that if you in the settings had selected for it to always use the video type Movie it became Home Movie. I am sorry about that.
